db/migrate/20221026151111_add_users_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddUsersToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true
+  end
```

Comment:
- [ ] Same here, Please add the foreign key between comments and users tables as `author_id`, not `user_id` as mentioned on the ERD table.